web-client/src/presenter/actions/updateMessageModalAttachmentsAction.test.ts

```diff
@@ -312,4 +314,4 @@ describe('updateMessageModalAttachmentsAction', () => {
       mockDocumentTitle,
     );
   });
-});
+});
```

Comment:
We like the newlines at the end of files and don't want to remove them.

```suggestion
});

```